src/dstack/_internal/core/backends/gcp/compute.py

```diff
@@ -70,14 +73,80 @@ def get_offers(
 
     def terminate_instance(
         self, instance_id: str, region: str, backend_data: Optional[str] = None
-    ):
+    ) -> None:
         try:
             self.instances_client.delete(
                 project=self.config.project_id, zone=region, instance=instance_id
             )
         except google.api_core.exceptions.NotFound:
             pass
 
+    def create_instance(
```

Comment:
As with AWS, logic is duplicated in `create_instance()` and `run_job()`. Should be refactored to avoid duplication.

Reply:
I've removed logic duplication.